SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+The GeoNetwork community takes the security of the software and all services based on the software product seriously. On this page you can find the versions for which the community provides security patches. 
+
+If you believe you have found a security vulnerability in the software or an implementation of the software, please report it to geonetwork@osgeo.org as described below. Do not publish the vulnerability in any public forums.
+
+## Supported Versions
+
+Each GeoNetwork release is supported with bug fixes for a limited period, with patch releases made approximately every three to six  months. We recommend to update to latest patch release as soon as possible to patch any vulnarabilities.
+
+| Version     | Supported          | Comment                 |
+| ----------- | ------------------ |------------------------ |
+| 4.0         | :white_check_mark: | Current version         |
+| 3.10.x      | :white_check_mark: | Supported up to 6 months after release of 3.12 |
```

Comment:
I don't think you should indicate 6 months because there is no formal commitment; and you mention a specific version `3.12` (so you would be updating this page often).